h5p_proplist_test.go

```diff
@@ -168,3 +245,21 @@ func compare(ds0, ds1 []float64) error {
 	}
 	return nil
 }
+
+func checkChunkCahche(nslots, nbytes int, w0 float64, dapl *PropList) error {
+	nslots_, nbytes_, w0_, err := dapl.GetChunkCache()
```

Comment:
not too fond of the `xxx_` convention.
perhaps postfix `Cache` instead? or something along these lines?

Reply:
Fixed